packages/markdown-render-engine/README.md

```diff
@@ -0,0 +1,57 @@
+# Markdown render engine
+
+## Overview
+
+This render engine is based on [react-markdown](https://github.com/rexxars/react-markdown) component.
+
+## Installation
+
+- by [`npm`](https://www.npmjs.com/)
+  ``` bash
+  npm i @kyma-project/dc-markdown-render-engine
+  ```
+
+- by [`yarn`](https://yarnpkg.com/en/)
+  ``` bash
+  yarn add @kyma-project/dc-markdown-render-engine
+  ```
+
+## Options
+
+> **NOTE**: This render engine support all options available in [react-markdown](https://github.com/rexxars/react-markdown#options) component.
+
+| Name | Required | Type | Default value | Description |
+|---|---|---|---|---|
+| `parsers` | No | [`MarkdownParserPlugin[]`](./src/types.ts#L24) | `[]` | Custom parsers for custom content in markdown files. More information about write parsers [here](https://github.com/aknuds1/html-to-react#with-custom-processing-instructions). |
+| `customRenderers` | No | [`Renderers`](./src/types.ts) | `{}` | An object where the keys represent the node type and the value is a React component. The object is merged with the default renderers. The props passed to the component varies based on the type of node. See [default](./src/renderers) implementations of renderers. |
+| `highlightTheme` | No | `any` | `{}` | Custom styles for highlight of code block. Component use for highlight [`PrismJS`](https://github.com/PrismJS/prism). See example [themes](https://github.com/PrismJS/prism-themes/tree/master/themes). |
+| `headingPrefix` | No | `string or ((source: Source) => string)`, where [`Source`](../documentation-component/src/interfaces/Source.ts) is type | `""` | Prefix for any heading appearing in markdown file. |
+| `copyButton` | No | `ReactNode` | `null` | Custom copy button in code blocks. Component accept only `code` as `string` to copy as prop.  |
```

Comment:
```suggestion
| `copyButton` | No | `ReactNode` | `null` | Defines a custom copy button in code blocks.  |
```
We need to rephrase this: `Component accepts only `code` as `string` to copy as prop. `